mxcubeweb/core/models/configmodels.py

```diff
@@ -104,11 +104,31 @@ class UISampleViewVideoControlsModel(UIPropertiesModel):
     ]
 
 
+class UISessionPickerTabModel(BaseModel):
+    id: Literal[
+        "active_tab", "scheduled_tab", "other_beamlines_tab", "all_sessions_tab"
```

Comment:
Maybe this for consistency, also this would match what is in the documentation:

```suggestion
        "active", "scheduled", "other_beamlines", "all"
```

Reply:
Done, but I've kept name "all_sessions" instead of just "all" - the latter sounds like it were to just open all the possible tabs